README.md

```diff
@@ -37,6 +37,10 @@ are `systemd-devel`, `json-c-devel`, and `libcurl-devel` on RPM-based distros,
 and `libjson-c-dev`, `libsystemd-journal-dev`/`libsystemd-dev` and
 `libcurl-*-dev` on Debian-based distros.
 
+To build from Git on RPM-based distros you'll need `autoreconf`, `automake`
+and `libtool` packages.
```

Comment:
Ah, wait. This is valid not only for RPM-based distros, but for Debian-based as well. So, please remove "on RPM-based distros". Also, the package name is `autoconf`, NOT `autoreconf`.

Reply:
Oh, yes. Also,for Debian pkg-config is needed, right?

Reply:
Yes.

Reply:
On Debian, pkg-config is needed even for building from a tarball.